docker-compose.yml

```diff
@@ -55,6 +66,7 @@ services:
       - SYSTEM_CLIENT_ID=system-client-id
       - SYSTEM_CLIENT_SECRET=system-client-secret
       - SESSION_SECRET=session-secret
+      - OS_PLACES_API_URL=http://localhost:9092/search/places/v1
```

Comment:
This should be `http://fake-os-places-api:9092/search/places/v1` now if you want it to hit the container

Reply:
👍